testing-with-jest.test.js

```diff
@@ -9,19 +9,73 @@
 // --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
 
 
+describe("addThemUpNo", () => {
+    test( "add two numbers" , () => {
+      expect(addThemUp()).toEqual(16)
```

Comment:
Almost! We need our expect to take in two numbers. So we would give it test numbers ourself, such as 10, 5.
`expect(addThemUp(10, 5)).toEqual(16)`